src/main/java/vg/civcraft/mc/civmodcore/command/PlayerCommand.java

```diff
@@ -2,50 +2,83 @@
 
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
+
+import vg.civcraft.mc.civmodcore.util.TextUtil;
+
+import org.bukkit.command.CommandSender;
 import org.bukkit.command.PluginCommand;
 
 public abstract class PlayerCommand implements Command {
 
 	private String name = "";
-
 	private String description = "";
-
 	private String usage = "";
-
 	private String identifier = "";
-
 	private int min = 0;
-
 	private int max = 0;
+	private boolean senderMustBePlayer = false;
+	private boolean errorOnTooManyArgs = true;
+	private CommandSender sender;
+	private String[] args;
```

Comment:
Yeah you can't really see the benefit with this PR alone. By passing the arguments to the base class it allows an inheriting class to generate methods likes:
    
getArgAsPlayer(int index)
getArgAsGroup(int index)

That's what I did for CivChat but those could probably be added to the PlayerCommand class itself.